app/src/contexts/Order/Order.tsx

```diff
@@ -294,14 +294,14 @@ const Order: React.FC = (props) => {
         break
     }
 
-    let approvalTxs: any[] = []
```

Comment:
I should get that in my prettier settings

Reply:
ill send over my config 🤟